end_to_end/test_checkpointing.sh

```diff
@@ -35,14 +35,15 @@ then
     CMD_DATA=" grain_worker_count=0 dataset_type=grain grain_train_files=/tmp/gcsfuse/array-record/c4/en/3.0.1/c4-train.array_record*"
 fi
 
-#Train
+#Train and save checkpoint
 CMD1="python3 -m MaxText.train MaxText/configs/base.yml run_name=$RUN_NAME steps=5 max_target_length=128 per_device_batch_size=1\
     metrics_file=saved_metrics.txt checkpoint_period=3 base_output_directory=$OUTPUT_PATH dataset_path=$DATASET_PATH\
     async_checkpointing=$ASYNC_CHECKPOINTING collect_stack_trace=$COLLECT_STACK_TRACE attention=$ATTENTION"
 CMD1+=$model_params
 CMD1+=$CMD_DATA
 
-CMD2="python3 -m MaxText.train MaxText/configs/base.yml run_name=$RUN_NAME steps=5 max_target_length=128 per_device_batch_size=1\
+#Restore checkpoint and continue training from the restored checkpoint
```

Comment:
Nit: space after #, also in the comment above (our linter is currently disabled) 

Could you explain here about the number of steps needing to be larger than the one before the checkpoint (as in the excellent PR description). Thank you!